django/core/management/commands/sqlmigrate.py

```diff
@@ -53,15 +54,15 @@ def handle(self, *args, **options):
         except KeyError:
             raise CommandError("Cannot find a migration matching '%s' from app '%s'. Is it in INSTALLED_APPS?" % (
                 migration_name, app_label))
-        targets = [(app_label, migration.name)]
+        target = (app_label, migration.name)
 
         # Show begin/end around output for atomic migrations, if the database
         # supports transactional DDL.
         self.output_transaction = migration.atomic and connection.features.can_rollback_ddl
 
         # Make a plan that represents just the requested migrations and show SQL
         # for it
-        plan = [(executor.loader.graph.nodes[targets[0]], options['backwards'])]
+        plan = [(executor.loader.graph.nodes[target], options['backwards'])]
```

Comment:
This small optimization is not related with fix. Merged in e12fea24f06f8911ddc2af1d6cbfb1adb529c1f2.